cypress/e2e/read-receipts/high-level.spec.ts

```diff
@@ -795,13 +794,10 @@ describe("Read receipts", () => {
                 // Then the room becomes unread
                 assertUnread(room2, 1);
             });
-            // XXX: fails because on CI we get a dot, but locally we get a count. Must be a timing issue.
-            it.skip("A room with an edit is still unread after restart", () => {
+            it("A room with an edit is still unread after restart", () => {
                 // Given a message is marked as read
-                goTo(room1);
+                goTo(room2);
```

Comment:
I R confoosed. If I am viewing room 2 and receive a message there, it is unread.
But an edit leads to an unread?

Should the test go to room 1 after the first `assertRead`?

Reply:
Ooh, yeah good point. But then have we found a bug, since this passes?

Reply:
I added a test in f208b70cbf, which passes, but I marked this one as not passing, because after your suggested change this test is now triggereng the existing bug we have about having an unread dot instead of a number.